src/devices/Rtc/Devices/Ds3231/Ds3231.cs

```diff
@@ -2,10 +2,10 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Device.I2c;
 using Iot.Device.Common;
 using Iot.Units;
+using System;
```

Comment:
please revert

Reply:
Why? Without it, it can't resolve `DateTime`.